Userland/Utilities/shot.cpp

```diff
@@ -39,10 +39,18 @@ int main(int argc, char** argv)
     Core::ArgsParser args_parser;
 
     String output_path;
+    const char* start_x = nullptr;
+    const char* start_y = nullptr;
+    const char* width = nullptr;
+    const char* height = nullptr;
     bool output_to_clipboard = false;
     int delay = 0;
 
     args_parser.add_positional_argument(output_path, "Output filename", "output", Core::ArgsParser::Required::No);
+    args_parser.add_option(start_x, "Pixels from where the screenshot gets taken (x)", "starting-x", 'x', "pixels");
+    args_parser.add_option(start_y, "Pixels from where the screenshot gets taken (y)", "starting-y", 'y', "pixels");
+    args_parser.add_option(height, "Screenshot height in pixels", "height", 'h', "pixels");
+    args_parser.add_option(width, "Screenshot width in pixels", "width", 'w', "pixels");
```

Comment:
This feels very clunky. It would be a lot nicer if you would provide the screenshot rect as a string e.g `shot -r 100,100:200x300` or similar.